taxonomies/categories.txt

```diff
@@ -1136,19 +1245,43 @@ fr:Bières sans gluten
 hu:Gluténmentes sör
 it:Birre senza glutine
 
+<en:Beers
+en:Special beers, Special beers with 5° alcohol, Special beers with 6° alcohol
+fr:Bières spéciales à 5° d'alcool, Bières spéciales à 6° d'alcool
+ciqual_food_code:en:5010
+ciqual_food_name:en:Beer, special (5-6° alcohol)
+ciqual_food_name:fr:Bière spéciale (5-6° alcool)
+
+<en:Beers
+en:5% beer, Beer with 5° alcohol, Beer with 4° alcohol
+fr:Bière à 4-5° d'alcool, Bière à 4%, Bière à 5%
+ciqual_food_code:en:5001
+ciqual_food_name:en:Beer, regular (4-5° alcohol)
+ciqual_food_name:fr:Bière coeur de marché (4-5° alcool)
+
+<en:Beers
+en:Beers with low alcohol-content, 3% beers, Beers with 3° alcohol
+fr:Bières faiblement alcoolisées, Bières à 3° d'alcool, Bières à 3%
+ciqual_food_code:en:5008
+ciqual_food_name:en:Beer, low alcohol-content (3° alcohol)
+ciqual_food_name:fr:Bière faiblement alcoolisée (3° alcool)
+
 <en:Non-Alcoholic beverages
-en:Non-alcoholic beers, non-alcoholic beer, Alcohol-free beer
+en:Non-alcoholic beers, non-alcoholic beer, Alcohol-free beer, Alcohol-free beers, Beers with less than 1,2° alcohol
 de:Alkoholfreie Biere, Alkoholfreies Bier
 es:Cervezas sin alcohol
 fi:alkoholittomat oluet
-fr:Bières sans alcool, bière sans alcool
+fr:Bières sans alcool, bière sans alcool, Bières à moins d'1,2° d'alcool
```

Comment:
@stephanegigandet how do we handle the comma ?

Reply:
"Bières à moins d'1.2° d'alcool", ça passerait ?

Reply:
oui

Reply:
j'ai rajouté un fix en anglais aussi. thx